docs/1_how_it_works/1_2_difference.md

```diff
@@ -0,0 +1,45 @@
+# Difference between Finetuner and other frameworks
+
+There are several fancy machine learning libraries out there,
+so what makes Finetuner unique?
+
+## Care about the quality of embeddings
+
+Finetuner is not designed to improve classification,
```

Comment:
I personally like this way: "Finetuner is not aiming at boosting the performance of classification ... **directly**"

Reply:
Since embeddings can be used in any downstream tasks